packages/openactive-integration-tests/test/helpers/flow-helper.js

```diff
@@ -89,60 +92,22 @@ class FlowHelper {
 
   P = pMemoize(async () => {
     throw new Error('This should not be called');
-    // this._assertStageShouldNotBeSkipped('P');
```

Comment:
Same again. By removing the obsolete RequestHelper.getOrder function, I had to remove its callees as well (and so on)

Reply:
Great to see, shall we just bin this code also?

Is there a reason why we couldn't go further and delete `BAfterP` etc. if they have no references in the code base?

Reply:
This is deleted. I intend to go further for flow-helper & request-state once some other stuff is done